docs/README.md

```diff
@@ -38,181 +38,92 @@ Code | Text | Description
-
-```
-checkout-account:1234\n
-checkout-algorithm:SHA512\n
-checkout-method:POST\n
-checkout-nonce:1234\n
-checkout-timestamp:2018-07-05T11:19:25.950Z\n
-REQUEST BODY
-```
-
-These are passed to the HMAC function which uses SHA256 algorithm. Carriage returns (\r) should not be used, only line feed (\n).
+* All `checkout-` headers in alphabetical order. The header keys must be in lowercase. Each header key and value are separated with `:`
+* HTTP body, or empty string if no body
 
+The headers are:
 
 field | info | description
 --- | --- | ---
-checkout-account | numeric | Checkout account ID
-checkout-algorithm | string | Used algorithm, either SHA256 or SHA512
-checkout-method | string | HTTP verb of request
+checkout-account | numeric | Checkout account ID, eg. 375917
+checkout-algorithm | string | Used signature algorithm, either sha256 or sha512
+checkout-method | string | HTTP verb of request, either GET or POST
 checkout-nonce | string | Unique identifier for this request
 checkout-timestamp | string | ISO 8601 date time
+checkout-transaction-id | string | Checkout transaction ID when accessing single transaction - not required for new payment request
 
-The HTTP verb, nonce and timestamp are used to mitigate various replay and timing attacks.
-
-### HMAC example (node.js)
+The HTTP verb, nonce and timestamp are used to mitigate various replay and timing attacks. Below is an example of the payload passed to a HMAC function:
 
-```javascript
-const crypto = require('crypto');
-
-const ACCOUNT = '375917';
-const SECRET = 'SAIPPUAKAUPPIAS';
-
-const headers = {
-  'checkout-account': ACCOUNT,
-  'checkout-algorithm': 'SHA256',
-  'checkout-method': 'POST',
-  'checkout-nonce': '564635208570151',
-  'checkout-timestamp': '2018-07-06T10:01:31.904Z'
-};
-
-const body = {
-  stamp: 'should-be-unique-for-merchant',
-  reference: '3759170',
-  amount: 1525,
-  currency: 'EUR',
-  language:'FI',
-  items: [
-    {
-      unitPrice: 1525,
-      units: 1,
-      vatPercentage: 24,
-      productCode: '#1234',
-      deliveryDate: '2018-09-01'
-    }
-  ],
-  customer: {
-    email: 'test.customer@example.com'
-  },
-  redirectUrls: {
-    success: 'https://ecom.example.com/cart/success',
-    cancel: 'https://ecom.example.com/cart/cancel'
-  }
-};
-
-const hmacPayload =
-  Object.keys(headers)
-    .sort()
-    .map((key) => [ key, headers[key] ].join(':'))
-    .concat(JSON.stringify(body))
-    .join("\n");
-
-// Expected HMAC: e6ed7ec0889db888f3067feb57e0a831b88da547902cd4f40ecb646d2bb763ac
-const hmac = crypto
-  .createHmac('sha256', SECRET)
-  .update(hmacPayload)
-  .digest('hex');
+```
+checkout-account:1234\n
+checkout-algorithm:sha256\n
+checkout-method:POST\n
+checkout-nonce:1234\n
+checkout-timestamp:2018-07-05T11:19:25.950Z\n
+REQUEST BODY
 ```
 
+See also code examples [HMAC calculation (node.js)](/examples#hmac-calculation-node-js) and [HMAC calculation (PHP)](/examples#hmac-calculation-php).
```

Comment:
there could maybe be a preposition after "code examples"? like code examples at or code examples of or something?